CI is failing due to OOM

The default ubuntu runner in github actions has only 7GB of RAM. 

- ref: About GitHub-hosted runners - GitHub Docs https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources 

In the CI, the Stable Diffusion weights are loaded, but it looks like OOM is occurring at this time.

```shell
============================= test session starts ==============================
platform linux -- Python 3.10.7, pytest-7.1.3, pluggy-1.0.0
rootdir: /home/runner/work/training-free-structured-diffusion-guidance/training-free-structured-diffusion-guidance
collected 40 items

/home/runner/work/_temp/cefdbf2f-5235-41fd-b534-a5dd91[31](https://github.com/shunk031/training-free-structured-diffusion-guidance/actions/runs/3218042736/jobs/5262058599#step:9:32)e216.sh: line 2:  1794 Killed                  pytest --color=yes -rf
tests/pipelines/tfsdg_pipeline_test.py 
Error: Process completed with exit code 137.
```
- ref: https://github.com/shunk031/training-free-structured-diffusion-guidance/actions/runs/3218042736/jobs/5262058599

--- a/tests/pipelines/tfsdg_pipeline_test.py
+++ b/tests/pipelines/tfsdg_pipeline_test.py
@@ -86,8 +86,12 @@ def _compare_generated_images(img1: Image.Image, img2: Image.Image) -> None:
 
 def test_cross_attention(model_name: str):
 
-    proposed_pipe = TFSDGPipeline.from_pretrained(model_name)
-    original_pipe = StableDiffusionPipeline.from_pretrained(model_name)
+    proposed_pipe = TFSDGPipeline.from_pretrained(
+        model_name, revision="fp16", torch_dtype=torch.float16
+    )
+    original_pipe = StableDiffusionPipeline.from_pretrained(
+        model_name, revision="fp16", torch_dtype=torch.float16
+    )
 
     _compare_cross_attention(
         proposed_module=proposed_pipe.unet,

